packages/elements/src/components/ino-snackbar/ino-snackbar.tsx

```diff
@@ -35,10 +35,15 @@ export class Snackbar implements ComponentInterface {
    */
   @Prop() actionText?: string = '';
 
+  /**
+   * Controls if Snackbar will appear at the top or at the bottom of the screen
+   */
+  @Prop() verticalAlignment?: 'top' | 'bottom' = 'top';
+
   /**
    * Controls if Snackbar is centered or left-aligned or right-aligned.
    */
-  @Prop() alignment: 'left' | 'right' | 'center' = 'center';
+  @Prop() horizontalAlignment: 'left' | 'right' | 'center' = 'right';
```

Comment:
` 'center' = 'right'` ?

Reply:
Yes, according to the designs, `right` should be the new default value.

Reply:
Oh, now I can see it 🙈 I thought that 'right' was assigned to 'center', so never mind 😂

Reply:
All made changes are good.  By testing it via storybook I recordnize that multiply instances were opened on clicking the button. I think the event target are not decoupled. Should I open an issue for that?